jax/_src/numpy/fft.py

```diff
@@ -444,15 +444,162 @@ def _fft_core_2d(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return _fft_core(func_name, fft_type, a, s, axes, norm)
 
 
-@implements(np.fft.fft2)
 def fft2(a: ArrayLike, s: Shape | None = None, axes: Sequence[int] = (-2,-1),
          norm: str | None = None) -> Array:
+  r"""Compute a two-dimensional discrete Fourier transform along given axes.
+
+  JAX implementation of :func:`numpy.fft.fft2`.
+
+  Args:
+    a: input array.
+    s: sequence of integers. Specifies the shape of the result. ``s`` should have
+      same length as ``axes``. If not specified, it will default to the shape of
+      ``a`` along the specified ``axes``.
+    axes: sequence of integers, default=(-2,-1). Specifies the axes along which
+      the transform is computed. Length of ``axes`` must be equal to 2. If not
+      specified, the transform is computed along the last two axes.
+    norm: string. The normalization mode. "backward", "ortho" and "forward" are
+      supported.
+
+  Returns:
+    An array containing the two-dimensional discrete Fourier transform of ``a``.
+
+  See also:
+    - :func:`jax.numpy.fft.fft`: Computes a one-dimensional discrete Fourier
+      transform.
+    - :func:`jax.numpy.fft.ifft`: Computes a one-dimensional inverse discrete
+      Fourier transform.
+    - :func:`jax.numpy.fft.ifft2`: Computes a two-dimensional inverse discrete
+      Fourier transform.
+    - :func:`jax.numpy.fft.fftn`: Computes a multidimensional discrete Fourier
+      transform.
+    - :func:`jax.numpy.fft.ifftn`: Computes a multidimensional inverse discrete
```

Comment:
I would remove `ifftn` and `ifft` from this list. I think of them as two-steps removed from this function.